mapswipe_workers/mapswipe_workers/firebase_to_postgres/transfer_results.py

```diff
@@ -92,9 +93,18 @@ def results_to_file(results, projectId):
     logger.info(f'Got %s groups for project {projectId} to transfer' % len(results.items()))
     for groupId, users in results.items():
         for userId, results in users.items():
-            timestamp = results['timestamp']
-            start_time = results['startTime']
-            end_time = results['endTime']
+
+            # check if all attributes are set, if not don't transfer the results for this group
```

Comment:
Why would data not exist?

Reply:
We talked about it with Benni yesterday, after sentry spiked with 2.8k errors in under 24h, because of one result object that has no `result` attribute. We've not been able to understand why this happened, but it did happen only once in about 2 weeks of operation. The incoming task does not seem to be the issue, as at least one other user provided correctly formatted results for it. So I think that checking and logging on both sides (JS+python) is probably a good approach for now, as it might help identify where things go wrong, if it ever happens again.

Reply:
basicalll we do checks on JS to prevent the real time user stats to be wrong, and in python to prevent storing wrong data in the postgres database